eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,11 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- Ignore System.Text.JSon 7.0.1 and dependencies until we can add
```

Comment:
Would be IMO good to open a tracking issue for this and add it to the comment prior to merging.

Reply:
> Would be IMO good to open a tracking issue for this and add it to the comment prior to merging.

Yeah, meant to do that originally - working on it now.

Reply:
Fixed in https://github.com/dotnet/deployment-tools/pull/266/commits/eaff4f995a881cd6442de7dfaf93e86340482fe3